a4kSubtitles/download.py

```diff
@@ -140,6 +164,7 @@ def download(core, params):
     __postprocess(core, filepath, lang_code)
 
     if core.api_mode_enabled:
+        __copy_sub_local(core, filepath)
```

Comment:
What is the reason to enable this only for the auto-download and not also for the manual sub search?

Reply:
For some reason I removed the comment I added there :).

Kodi it self will handle this process for manual search.